manifests/enterprise/repo/apt.pp

```diff
@@ -0,0 +1,29 @@
+class sensu::enterprise::repo::apt {
```

Comment:
Is there a reason this and sensu::enterprise::repo::yum couldn't be merged into sensu::repo::yum/apt?  It'd just be a conditional in them to set the url based on enterprise true/false, right?